app/views/categories/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<main>
+<div class="main-page">
+  <%= form_with model: @category, url: user_categories_path(current_user),  method: :post do |form| %>
+    <% if @category.errors.any? %>
+      <div style="color: red">
+        <h2><%= pluralize(@category.errors.count, "error") %>
+          prohibited category from being saved:</h2>
+
+        <ul>
+          <% @category.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+
+    <div class="form">
+    <div class="field">
+      <%= form.text_field :name, placeholder: "Name", class:"input" %>
+    </div>
+
+    <div class="field">
+      <%= form.file_field :icon, placeholder: "Upload icon" %>
+    </div>
+
+    <div class="actions">
+      <%= form.submit class:"btn" %>
+    </div>
+    </div>
+  <% end %>
+  </div>
+</main>
```

Comment:
- [ ] The new category form should have a back button "<" that takes me back to the categories page when I click it.

  ![image](https://github.com/Kaghenimbale/budget-app/assets/36542986/230dd96a-51bc-497d-813b-f764ec94e212)